### PR TITLE
feat: add support for common labels in helm charts

### PR DESCRIPTION
**Pull request by electrosenpai**
Signed-off-by: Jean-Baptiste DONNETTE <jean-baptiste.donnette@payfit.com>

When we used a policies manager as Kyverno, it's usefull to be able to add a common labels on all resources created by the chart in order to respect the policy.

### CODE REVIEW

File: src/chartmuseum/templates/_helpers.tpl [hunk 1]
```diff
@@ -54,6 +54,9 @@ Selector labels
 {{- define "chartmuseum.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "chartmuseum.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Values.commonLabels}}
```
Comment (cbuto) [inline src/chartmuseum/templates/_helpers.tpl:57]: can we move these new common labels under "chartmuseum.labels"?

Comment (electrosenpai) [reply src/chartmuseum/templates/_helpers.tpl:57]: As long as we have a way to deploy label on all ressources sounds good for me !

Comment (electrosenpai) [reply src/chartmuseum/templates/_helpers.tpl]: Should be good @cbuto

File: src/chartmuseum/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Host your own Helm Chart Repository
 name: chartmuseum
-version: 3.9.0
+version: 3.9.1
```
Comment (cbuto) [inline src/chartmuseum/Chart.yaml:4]:
```suggestion
version: 3.9.2
```

File: src/chartmuseum/templates/_helpers.tpl [hunk 3]
```diff
@@ -45,6 +45,9 @@ helm.sh/chart: {{ include "chartmuseum.chart" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
+{{- if .Values.commonLabels}}
```
Comment (cbuto) [inline src/chartmuseum/templates/_helpers.tpl:48]: looking at the chart resources, it looks like the PV and PVC resources don't use these common labels so they won't pick these up. 

If you want the PV and PVC to be labeled, we'll need to update those with:

```
  labels:
    {{- include "chartmuseum.labels" . | nindent 4 }}
```

**Review by cbuto — COMMENTED** [review]
lgtm after the chart version is bumped.

I left one other comment that we can tackle now or later

**Review by cbuto — APPROVED** [review]

**Comment by cbuto** [comment]
thanks @electrosenpai! 🚀

**Pull request merged**